apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -162,9 +152,8 @@ Your `.env` file should resemble the following:
 # PostgreSQL connection string used for migrations
 DIRECT_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres"
 # PostgreSQL connection string with pgBouncer config — used by Prisma Client
-DATABASE_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres?connection_limit=1"
+DATABASE_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres?pgbouncer=true&connection_limit=1"
```

Comment:
The port of the direct URL and the PgBouncer URL need to be different - otherwise they are using the same connection pooled connection (which will crash and burn).

Reply:
Ah, my bad. it's a copy-paste mistake. thanks for highlighting that.